### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by keex1k**
- [DEMO LINK](https://keex1k.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://keex1k.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,8 +6,36 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card">
```
Comment (volodymyr-soltys97) [inline src/index.html:12]:
```suggestion
    <div class="card">
```

File: src/index.html [hunk 2]
```diff
@@ -6,8 +6,36 @@
...
+      <div class="card__image"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:13]: Add the image with the img tag, not as a background

File: src/index.html [hunk 3]
```diff
@@ -6,8 +6,36 @@
...
+
+      <section class="card__info">
```
Comment (volodymyr-soltys97) [inline src/index.html:15]: Redundant
```suggestion
```

File: src/index.html [hunk 4]
```diff
@@ -6,8 +6,36 @@
...
+        <div class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__code">Product code: 195434</div>
```
Comment (volodymyr-soltys97) [inline src/index.html:17]: Use the semantic tag for text everywhere

File: src/index.html [hunk 5]
```diff
@@ -6,8 +6,36 @@
...
+      </section>
+
+      <section class="card__rate">
+        <section class="card__stars">
```
Comment (volodymyr-soltys97) [inline src/index.html:21]:
```suggestion
      <div class="card__rate">
        <div class="stars stars--4">
```

File: src/index.html [hunk 6]
```diff
@@ -6,8 +6,36 @@
...
+          <div class = "card__star"></div>
+          <div class = "card__star"></div>
+          <div class = "card__star"></div>
+          <div class = "card__star"></div>
+          <div class = "card__star"></div>
+        </section>
+        <div class="card__reviews">Reviews: 5</div>
+      </section>
+
+      <section class="card__price">
```
Comment (volodymyr-soltys97) [inline src/index.html:31]:
```suggestion
      <div class="card__price">
```

File: src/index.html [hunk 7]
```diff
@@ -6,8 +6,36 @@
...
+        <div class="card__text">Price:</div>
+        <div class="card__number">$2,199</div>
+      </section>
+
+      <section>
```
Comment (volodymyr-soltys97) [inline src/index.html:36]: Redundant
```suggestion
```

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,3 +1,163 @@
+$card-width: 200px;
+$imac-width: 160px;
+$imac-margin: ($card-width - $imac-width) / 2;
+$desc-width: 166px;
+$desc-margin: ($card-width - $desc-width) / 2;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:5]: Move variables to the separate file

File: src/styles/index.scss [hunk 9]
```diff
@@ -1,3 +1,163 @@
+$card-width: 200px;
+$imac-width: 160px;
+$imac-margin: ($card-width - $imac-width) / 2;
+$desc-width: 166px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:4]: No need to hardcode width for the content, use 100% instead
```suggestion
$desc-width: 100%;
```

File: src/styles/index.scss [hunk 10]
```diff
@@ -1,3 +1,163 @@
...
+      flex-shrink: 0;
+
+      :last-child {
+        margin-right: 4px;
+      }
+
+      :first-child {
+        margin-left: 0;
+      }
+    }
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:87]: Move stars styles to the separate file

File: src/styles/index.scss [hunk 11]
```diff
@@ -1,3 +1,163 @@
...
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__text {
+    color: #616070;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:117]: Default value, so you can just remove it
```suggestion
```

File: src/styles/index.scss [hunk 12]
```diff
@@ -1,3 +1,163 @@
...
+    line-height: 18px;
+  }
+
+  &__number {
+    color: #060b35;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:122]: Create the variable for this color and use it everywhere

File: src/styles/index.scss [hunk 13]
```diff
@@ -1,3 +1,163 @@
...
+    justify-content: center;
+    height: 40px;
+    margin: 16px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    color: #fff;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px; /* 114.286% */
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:142]:
```suggestion
    line-height: 16px; 
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your styles doesn't work on the demo link, you need to make a deploy again
image

**Comment by keex1k** [comment]
> Hi, your styles doesn't work on the demo link, you need to make a deploy again
> image

I deployed again and nothing. Can you tell me what is wrong?

**Pull request closed without merging**